### PR TITLE
ldb: cleanup code according to Go style

### PR DESCRIPTION
**Pull request by edganiukov**
I have cleaned a bit code, according to Go style.

### CODE REVIEW

File: lexer/tokenizer.go [hunk 1]
```diff
@@ -49,9 +48,7 @@ func belongsTogether(current []rune, next rune) bool {
 		switch current[0] {
 		case '+':
 			return next == '+'
-		case '!':
-			fallthrough
-		case '=':
+		case '!', '=':
```
Comment (BenchR267) [inline lexer/tokenizer.go:51]: Ah great, that ist what I was searching for, but it wasn't used in any of the examples. So I thought it's not possible in Go :D

**Review by BenchR267 — COMMENTED** [review]
Thanks! Do you have any recommendation for a linter that could suggest those changes?

File: lexer/tokenizer.go (hunk 1 above)
Comment (edganiukov) [reply lexer/tokenizer.go:51]: `fallthrough` is useful if you have additional logic for the next case, but in this case simple just write comma separated values

Comment (BenchR267) [reply lexer/tokenizer.go:51]: Sure, makes sense. thanks! 👍

**Pull request merged**